### PR TITLE
open cluster-policy-controller ports for metrics

### PR DESCRIPTION
**Pull request by atiratree**
This opens ports for gathering cluster-policy-controller metrics. The container itself is running on the same pod as kube-controller-manager.

I updated the configs in similar manner as kube-controller-manager. Please let me know if there are other steps needed.

### CODE REVIEW

**Comment by jstuever** [comment]
Is there an enhancement doc for this we can reference?

**Comment by atiratree** [comment]
@jstuever no there is not. Should I create one?

**Comment by atiratree** [comment]
@jstuever fyi it is also a part of a Jira issue: WRKLDS-308

**Comment by atiratree** [comment]
rebased

**Comment by atiratree** [comment]
> Is there an enhancement doc for this we can reference?

I just found out we have a doc describing this port: https://github.com/openshift/enhancements/blob/master/dev-guide/host-port-registry.md

we can add metrics to the notes there once this PR merges

File: data/data/aws/cluster/vpc/sg-master.tf [hunk 1]
```diff
@@ -289,6 +289,28 @@ resource "aws_security_group_rule" "master_ingress_kube_controller_manager_from_
   to_port   = 10257
 }
 
+resource "aws_security_group_rule" "master_ingress_cluster_policy_controller" {
```
Comment (staebler) [inline data/data/aws/cluster/vpc/sg-master.tf:292]: The AWS UPI should be updated with this as well. `upi/aws/cloudformation/03_cluster_security.yaml`

Comment (atiratree) [reply data/data/aws/cluster/vpc/sg-master.tf:292]: Thanks,  updated.

**Review by staebler — APPROVED** [review]
/lgtm

**Pull request merged**